lib/retrieval-stats.js

```diff
@@ -109,6 +118,11 @@ export const buildRetrievalStats = (measurements, telemetryPoint) => {
   for (const [result, count] of Object.entries(indexerResultBreakdown)) {
     telemetryPoint.floatField(`indexer_rate_${result}`, count / totalCount)
   }
+
+  telemetryPoint.floatField('rate_tasks_advertising_http', tasksQueryingIndexer.size > 0
```

Comment:
```suggestion
  telemetryPoint.floatField('rate_of_tasks_advertising_http', tasksQueryingIndexer.size > 0
```

At first I interpreted "rate" as "to rate something"

Reply:
What would be a better name? How about `http_availability_rate`?

Alternatively: `tasks_with_http_retrieval_available`. (I find this rather long.)

Reply:
What did you think of my suggestion?

Reply:
The name you suggested is a bit long, plus, as you wrote in the other thread, tasks are not advertising anything. 

So I guess a better name is `rate_of_tasks_with_http_advertisement`

I can also rename "task" to "deal", because that's where we want to arrive eventually - a task will be defined as "check if this deal is retrievable over HTTP?"


That gives us `rate_of_deals_advertising_http`, which sounds reasonable to me.

WDYT?

Reply:
good idea to move from tasks to deals. +1!